### PR TITLE
bugfix - missing account updates from accounts history tables

### PR DESCRIPTION
**Pull request by piotrkosecki**
Added balance updated account ids from the block header metadata.

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala [hunk 1]
```diff
@@ -682,9 +682,14 @@ class TezosNodeOperator(
       val proposalsMap = proposalsState.toMap
       fetchedBlocksData.map {
         case (offset, md) =>
+          val accountIdsWithUpdatedBalances = for {
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala]: It works, might it read better if extracted to a local method?

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala [hunk 2]
```diff
@@ -682,9 +682,14 @@ class TezosNodeOperator(
...
+            blockHeaderMetadata <- discardGenesis.lift(md.metadata).toList
+            balanceUpdate <- blockHeaderMetadata.balance_updates
+            id <- balanceUpdate.contract.map(_.id).orElse(balanceUpdate.delegate.map(_.value)).toList
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala]: Are we sure that `contract` and `delegate` are mutually exclusive in the update?
Maybe you ask me this already but I didn't get exactly your answer possibly.

We might better find a way to compose those "inclusively", like
`update.contract.map(_.id).toList ::: update.delegate.map(_.value).toList`

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala (hunk 1 above)
Comment (piotrkosecki) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala]: ok, extracted

File: src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala (hunk 2 above)
Comment (piotrkosecki) [reply src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala]: you are right, I  made this assumption on my own, changed

**Comment by ivanopagano** [comment]
I'm not sure we have test suites at that level of operations, if so, you might wanna add a test case.

**Pull request merged**